src/step_db.cpp

```diff
@@ -115,12 +115,14 @@ bool fields_chunk::step_db(field_type ft) {
           default: meep::abort("bug - non-cylindrical field component in Dcyl");
         }
 
-      STEP_CURL(the_f, cc, f_p, f_m, stride_p, stride_m,
-                gv, gv.little_owned_corner0(cc), gv.big_corner(),
-                Courant, dsig, s->sig[dsig],
-                s->kap[dsig], s->siginv[dsig], f_u[cc][cmp], dsigu, s->sig[dsigu], s->kap[dsigu],
-                s->siginv[dsigu], dt, s->conductivity[cc][d_c], s->condinv[cc][d_c],
-                f_cond[cc][cmp]);
+      for (size_t i = 0; i < gvs.size(); ++i) {
+        STEP_CURL(the_f, cc, f_p, f_m, stride_p, stride_m,
+                  gvs[i], gvs[i].little_owned_corner0(cc), gvs[i].big_corner(),
```

Comment:
The first argument on this line should be the original grid volume, `gv`, *not* `gvs[i]` — `STEP_CURL` needs to know "where" you are relative to the whole chunk volume in order to correctly index into the sigma arrays etcetera.

Reply:
Fixed.